workers/ohsome_quality_analyst/indicators/mapping_saturation/sigmoid_curve.py

```diff
@@ -1027,7 +1027,20 @@ def getBestFittingCurve(self, preprocessing_results: Dict) -> list:
         errorsListSingle = []
         if not math.isnan(initParamsSingle[3]):
             inits5curves = self.sortInits5curves(df1.li, df1.yValues)
-            if not math.isnan(inits5curves[3]):
+            # add up the 2 lists and then check each of the 4 values,
+            # if they are not nan
+            validity_result = all(  # check if all elements are true
+                map(
+                    math.isfinite,  # check if number is finite --> excludes nan
```

Comment:
Why do you use `isfinite` instead of the more explicit and easier to understand `ìsnan`?

Reply:
Sorry, missed previous discussion about this change.